src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentResourceIT.java

```diff
@@ -40,8 +40,7 @@
 import static io.restassured.RestAssured.given;
 import static io.restassured.http.ContentType.JSON;
 import static javax.ws.rs.core.Response.Status.OK;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.*;
```

Comment:
You might need to update your ide to prevent it adding wildcard imports, some details here, we don't use wildcard imports.
https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports